Email notifications for create,update,delete deployment (#182)
GoogleCloudPlatform/rad-lab: docs/docs/rad-lab-ui/troubleshooting.md

---

> # Troubleshooting
> 
> ### Email Server setup
> Email notifications are customizable,Admins can `enable` or `disable` email notifications on `Global Variables` setup.
> Inorder to proceed with email notifications admin has to store the email server (smtp server or a dedicated e-mail) credentials as part of `Global Variables`.
> 
> `MAIL SERVER EMAIL` will be stored in `firestore` and `MAIL SERVER PASSWORD` will be securly stored on `Google Secret Manager`
> 
> Please feel free to reach out and to create a [GitHub issue](https://github.com/GoogleCloudPlatform/rad-lab/issues) in case of any issues or concerns.

Reviewer: ```suggestion
# Troubleshooting

Please feel free to reach out and to create a [GitHub issue](https://github.com/GoogleCloudPlatform/rad-lab/issues) in case of any issues or concerns.

## Email Notifications
You can optionally enable RAD Lab notification for deployment events. This includes deployment creations, updates, and deletions.

If enabled, the following users/groups (defined in the module's Terraform variables) will receive email notifications:
- The individual taking the action
- `trusted_users`
- `trusted_groups`
- `owner_users`
- `owner_groups`

Currently only sending via gmail is supported. It is recommended to [create a new gmail address](https://support.google.com/mail/answer/56256?hl=en) for this purpose only. Do **not** enable two factor authentication (2FA) on this account. You will then provide this email and its password to RAD Lab UI via the `Global Variables` setup.

The email address will be store in Firestore, and email password will be securely stored in Google's [Secret Manager](https://cloud.google.com/secret-manager)

```

# Troubleshooting

Please feel free to reach out and to create a [GitHub issue](https://github.com/GoogleCloudPlatform/rad-lab/issues) in case of any issues or concerns.

> ## Email Notifications

Reviewer 2: I think the best place for this section is under  [Global Admin Variables](https://googlecloudplatform.github.io/rad-lab/docs/rad-lab-ui/ui_installation/publishing-modules/#global-admin-variables) section compared to Troubleshooting section.

And also we should remove the mention of `Preferred Google Cloud Region and Zone` from Global Admin Variables.

Author: Moved `Email Notifications` section to `Global Admin Variables`

You can optionally enable RAD Lab notification for deployment events. This includes deployment creations, updates, and deletions.

If enabled, the following users/groups (defined in the module's Terraform variables) will receive email notifications:
- The individual taking the action
- `trusted_users`
- `trusted_groups`
- `owner_users`
- `owner_groups`

> Currently only sending via gmail is supported. It is recommended to [create a new gmail address](https://support.google.com/mail/answer/56256?hl=en) for this purpose only and generate a `App Password` to authenticate it by followig [Sign in with app passwords](https://support.google.com/mail/answer/185833?hl=en)
> 

Reviewer: ```suggestion
Currently only sending via gmail is supported. It is recommended to [create a new gmail address](https://support.google.com/mail/answer/56256?hl=en) for this purpose only and generate an `App Password` to authenticate it by following [these directions](https://support.google.com/mail/answer/185833?hl=en).

```
